inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_ceiling.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "dsr_tests_common.hpp"
+
+#include <functional_test_utils/layer_test_utils.hpp>
+#include <ngraph_functions/builders.hpp>
+#include <vpu/ngraph/operations/dynamic_shape_resolver.hpp>
+
+namespace {
+
+using namespace LayerTestsUtils::vpu;
+
+using Parameters = std::tuple<
+    DataType,
+    DataShapeWithUpperBound,
+    LayerTestsUtils::TargetDevice
+>;
+
+class DSR_Ceiling : public testing::WithParamInterface<Parameters>, public DSR_TestsCommon {
+protected:
+    std::shared_ptr<ngraph::Node> createTestedOp() override {
+        const auto& parameters = GetParam();
+        const auto& inDataType = std::get<0>(parameters);
+        const auto& inDataShapes = std::get<1>(parameters);
+        targetDevice = std::get<2>(parameters);
+
+        const auto inputSubgraph = createInputSubgraphWithDSR(inDataType, inDataShapes);
+
+        const auto ceiling = std::make_shared<ngraph::opset3::Ceiling>(inputSubgraph);
```

Comment:
Oh, it seems like I was wrong when I said it's similar to clamp in our offline discussion. They are almost the same, but the reason why `Clamp` is extracted into separate test class is that it has additional parameters in its constructors unlike other postops (better to compare `Ceiling` with `Floor`).
So just put `Ceiling` operation into test instantiations in `dsr_unary_elementwise.cpp` & `dynamic_to_static_shape_unary_elementwise.cpp` test files.

Reply:
BTW, I don't see a unit test (that compares function vs function)

Reply:
Fixed